advanced/grids.md

```diff
@@ -98,12 +98,14 @@ Show audit logs.
 $ kontena grid audit-log
 ```
 
-## Show Kontena Platform Grid Environment Details
+## Show Kontena Node Configuration
 
-Outputs currently used Kontena Platform Grid environment variables that can be used to configure Kontena CLI.
+Generate the [`/etc/kontena-agent.env` environment variables](../references/environment-variables#kontena-agent) required when manually provisioning nodes using grid tokens:
```

Comment:
This links to the "Kontena Agent" section on the environment variables reference. Do we also want to avoid the mention of `/etc/kontena-agent.env` here?